contracts/src/libraries/FixedPointMath.sol

```diff
@@ -376,4 +376,11 @@ library FixedPointMath {
     function max(int256 a, int256 b) internal pure returns (int256) {
         return a > b ? a : b;
     }
+
+    /// @dev Calculates the absolute value of an int256.
+    /// @param a The value.
+    /// @return The absolute value of the input.
+    function abs(int256 a) internal pure returns (int256) {
+        return a < 0 ? -a : a;
```

Comment:
maybe i'm missing it, but is there a unit test for this?

Reply:
not seeing it either